packages/cli/package.json

```diff
@@ -18,33 +18,31 @@
     "celo-cli"
   ],
   "engines": {
-    "node": ">=14"
```

Comment:
from what i can tell the cli works well on node 16 so i see no reason to limit to 18 here